journal/journal-app.js

```diff
@@ -32,69 +31,121 @@ function addJournalNew(journalContainer, existing) {
 
     if (existing) {
-
         journalList.forEach(journal => {
-            journalID = createJournalElement(journal.id, journal.title, 
+            journalID = createJournalElement(journal.id, journal.title,
                 journal.documentation, journal.reflection, modalRef);
 
-                const journalElementTest = document.createElement('div');
-                journalElementTest.classList.add('journal-widget');
-                journalElementTest.textContent = 'Placeholder Title';
-                journalElementTest.setAttribute('widget-id', journalID);
-            
-                journalElementTest.addEventListener('dblclick', function(event) {
-                    document.getElementById('overlay').style.display = 'block';
-                    const currentJournalID = event.target.getAttribute('widget-id');
-                    const journalEntries = document.querySelectorAll('.journal-entry');
-                    hideOtherJournalEntries(journalEntries, currentJournalID);
-                    document.getElementById('modal').style.display = 'block';
-                });
-
-                journalContainer.append(journalElementTest);
+            // Creating journal widget for existing journal
+            createJournalWidget(journalContainer, journalID);
         });
-    }
-    else {
-        journalID = createJournalElement(journalTemplate.id, journalTemplate.title, 
+    } else {
+        journalID = createJournalElement(journalTemplate.id, journalTemplate.title,
             journalTemplate.documentation, journalTemplate.reflection, modalRef);
-            
-            const journalElementTest = document.createElement('div');
-            journalElementTest.classList.add('journal-widget');
-            journalElementTest.textContent = 'Placeholder Title';
-            journalElementTest.setAttribute('widget-id', journalID);
+
+        // Creating journal widget for new journal
+        createJournalWidget(journalContainer, journalID);
+
+        // Save new journal to localStorage
+        journalList.push(journalTemplate);
+        saveNotes(journalList);
+    }
+}
+
+// Extracted function to create and append the journal widget
+function createJournalWidget(journalContainer, journalID) {
+    const journalElementTest = document.createElement('div');
+    journalElementTest.classList.add('journal-widget');
+    // journalElementTest.textContent = 'Insert Title';
+    const journalWidgetTitle = document.createElement('span')
+    journalWidgetTitle.classList.add('journal-widget-title')
+    journalWidgetTitle.textContent = 'Insert Title';
+    journalElementTest.append(journalWidgetTitle);
+    journalElementTest.setAttribute('widget-id', journalID);
+
+    // Create a container for the buttons
+    var buttonContainer = document.createElement('div');
+    buttonContainer.style.position = 'absolute';
+    buttonContainer.style.top = '5px';
+    buttonContainer.style.right = '5px';
```

Comment:
Why do we modify the styling of the element here instead of the CSS file?

Reply:
moved it to css file